Composer/packages/ui-plugins/virtual-assistant-creation/src/components/index.ts

```diff
@@ -0,0 +1,3 @@
+// Copyright (c) Microsoft Corporation.
```

Comment:
Avoid barrel files (other index.ts exports) other than the top level one for the library.